lib/view_component/base.rb

```diff
@@ -104,7 +104,7 @@ def render_in(view_context, &block)
       before_render
 
       if render?
-        render_template_for(@__vc_variant).to_s + output_postamble
+        output_preamble + render_template_for(@__vc_variant).to_s + output_postamble
```

Comment:
Looks like tests are failing because this results in escaped HTML output. Try this instead:

```ruby
render_template_for(@__vc_variant).to_s
  .prepend(output_preamble)
  .concat(output_postamble)
```

Reply:
@camertron that resulted in a `FrozenError`. Got any other ideas?